conf/defaults.ini

```diff
@@ -230,6 +230,12 @@ snapshot_remove_expired = true
 # Number dashboard versions to keep (per dashboard). Default: 20, Minimum: 1
 versions_to_keep = 20
 
+#################################### Annotations ##################
+
+[annotations]
+# Number of days to keep annotations (per alert). Default: 60, Minimum: 5
+days_to_keep = 60
```

Comment:
I'm torn about having this on by default.

Reply:
I guess it should be turned off by default?

Reply:
I agree! At least now when it operates on all kinds of annotations. Maybe cleanup of alert annotations enabled per default would have made sense if it was the default from the beginning when alerting was introduced, but now it's sort of a breaking change so think we have to have it disabled per default.